<launch>
	
	<rosparam command="load" file="$(find al_prediction)/config/behavior_svm_models.yaml" />
	<node name="al_effect_predictor" pkg="al_prediction" type="effect_predictor" output="screen" respawn="false">
		<param name="models_directory_path" type="string" value="$(find al_prediction)/models"/>
	</node>
</launch>
